src/main/java/io/cdap/plugin/dlp/DLPTransformPluginConfig.java

```diff
@@ -186,4 +221,19 @@ public void validate(FailureCollector collector, Schema inputSchema) {
       }
     }
   }
+
+  public String getLocation() {
+    return location;
+  }
+
+  public LocationName getDlpParentResourceLocationUtil(String project, String location) {
```

Comment:
arguments that can be null should be annotated like `@Nullable String location`

Reply:
though this method doesn't need to take in any arguments. It should be getLocationName() and use getProject() and getLocation() inside the method.

Reply:
I have updated the function names to getLocationNameUtil(String project, String location) and getLocationName()
getLocationName() - uses getProject(), getLocation() and getLocationNameUtil(String project, String location)
getLocationNameUtil(String project, String location) - added this util function for the sake of testing, since we can't set the `project` variable in DLPTransformPluginConfig. If you see any better way to test this functionality, please suggest.

Reply:
Looking at this again, it seems to make more sense for getLocation() to do the null check and return global. Unless there's a reason we need to differentiate between a user entered null and user entered global?